projects/plugins/jetpack/modules/masterbar/admin-menu/load.php

```diff
@@ -7,6 +7,35 @@
 
 namespace Automattic\Jetpack\Dashboard_Customizations;
 
+use Automattic\Jetpack\Status;
+
+require_once __DIR__ . '/class-dashboard-switcher.php';
+
+/**
+ * Check if the current request is an API request or not.
+ *
+ * @return bool
+ */
+function is_api_request() {
```

Comment:
Just a helper function that was extracted from the current script to be able to reuse it.